src/GuppyDataExplorer/index.jsx

```diff
@@ -2,115 +2,92 @@ import React from 'react';
+      <React.Fragment>
+        <div className='guppy-explorer__tabs'>
+          {explorerConfig.map((element, index) => {
+            let tabTitle = '';
+            if (element.tabTitle) {
+              tabTitle = element.tabTitle;
+            } else if (element.guppyConfig && element.guppyConfig.dataType) {
+              tabTitle = capitalizeFirstLetter(element.guppyConfig.dataType);
+            }
+
+            return (
+              <React.Fragment key={index}>
+                <div
+                  className={'guppy-explorer__tab'.concat(this.state.tab === index ? ' guppy-explorer__tab--selected' : '')}
+                  onClick={() => this.onTabClick(index)}
+                  role='button'
+                  tabIndex={index}
+                >
+                  <h3>{tabTitle}</h3>
+                </div>
+              </React.Fragment>
+            );
+          })}
+        </div>
+      </React.Fragment>
+    );
+
     return (
       <div className='guppy-explorer'>
         {
-          config.fileExplorerConfig ? (
-            <div className='guppy-explorer__tabs'>
-              <div
-                className={'guppy-explorer__tab'.concat(this.state.tab === 0 ? ' guppy-explorer__tab--selected' : '')}
-                onClick={() => this.props.history.push('/explorer')}
-                role='button'
-                tabIndex={0}
-              >
-                <h3>Data</h3>
-              </div>
-              <div
-                className={'guppy-explorer__tab'.concat(this.state.tab === 1 ? ' guppy-explorer__tab--selected' : '')}
-                onClick={() => this.props.history.push('/files')}
-                role='button'
-                tabIndex={-1}
-              >
-                <h3>Files</h3>
-              </div>
-            </div>
-          ) : null
+          (explorerConfig.length > 1) ?
+            tabFragment
+            : null
         }
-        <div className={config.fileExplorerConfig ? 'guppy-explorer__main' : ''}>
+        <div className={(explorerConfig.length > 1) ? 'guppy-explorer__main' : ''}>
           <GuppyDataExplorer
-            adminAppliedPreFilters={guppyExplorerConfig[this.state.tab].adminAppliedPreFilters}
-            chartConfig={guppyExplorerConfig[this.state.tab].charts}
-            filterConfig={guppyExplorerConfig[this.state.tab].filters}
-            tableConfig={guppyExplorerConfig[this.state.tab].table}
-            heatMapConfig={this.state.tab === 0 ? config.dataAvailabilityToolConfig : null}
-            guppyConfig={{ path: guppyUrl, ...guppyExplorerConfig[this.state.tab].guppyConfig }}
+            adminAppliedPreFilters={explorerConfig[this.state.tab].adminAppliedPreFilters}
+            chartConfig={explorerConfig[this.state.tab].charts}
+            filterConfig={explorerConfig[this.state.tab].filters}
+            tableConfig={explorerConfig[this.state.tab].table}
+            heatMapConfig={this.state.tab === 0 ? dataAvailabilityToolConfig : null}
```

Comment:
this worries me a bit. The first tab won't always be the one where we want the heatmap, and we could want different heatmaps in multiple tabs. could we move the `dataAvailabilityToolConfig` to the `explorerConfig` items config in a backwards compatible way? 😬the heatmap will just be another kind of chart

Reply:
right! the `dataAvailabilityToolConfig` seems need some tweak. I'll fix it

Reply:
you can also create a ticket if it's a lot of work?

Reply:
I did a few tries, seems for now the easiest option is to put the `dataAvailabilityToolConfig` as a whole inside a single tab's config

It would be cool if we can just put `mainField`, `aggFields` etc in the `guppyConfig` for one tab. But right now it won't work because I think if you set `aggFields` for `guppyConfig` the guppy wrapper will preforms sub-aggregations, and that will breaks other UI components 🤷‍♂️ 

I can create a ticket for that future improvement

Reply:
right, the DAT is using a separate guppywrapper so i think it's normal that it has a separate guppyconfig? but a ticket is fine :)